docs/user-guide/side-bar.md

```diff
@@ -1,94 +1,91 @@
+The *Sidebar* is a navigation panel containing various elements that help the user to explore the map. In particular, it is possible zooming, changing the extent, navigating in 3D mode and querying objects on the map. 
+Moreover, the following icon <img src="../img/button/collapse.jpg" class="ms-docbutton"/> is used to expand/collapse the sidebar.
 
-<img src="../img/side-bar/side-bar-1.jpg" class="ms-docimage"/>
+<img src="../img/side-bar/sidebar.jpg" class="ms-docimage"/>
 
 Geolocation tool
 ----------------
 
-The Geolocation tool allows you to locate your real position on the map.
-
-* **Click** on the *Locate me* icon <img src="../img/button/geolocation.jpg" class="ms-docbutton"/>. The icon will be highlighted in green and your position will be shown on the map.
+Trought the *Show my position* <img src="../img/button/geolocation.jpg" class="ms-docbutton"/> the user can center the map on his position. Therefore the button turns green. 
 
 <img src="../img/side-bar/position.jpg" class="ms-docimage"/>
 
+The position is still active even when the user interacts with the map; with a single click on the button it is possible re-center the map on his position. 
+To desable the position the button needs to be duble clicked. 
+
 Zooming tools
 -------------
 
-[MapStore](https://mapstore.geo-solutions.it/mapstore/#/) provides several tools to zoom in and out, and to switch the map extent.
+[MapStore](https://mapstore.geo-solutions.it/mapstore/#/) provides several tools allows the user to:
 
-* You can increase the map zoom by using the zoom in icon <img src="../img/button/zoom-in.jpg" class="ms-docbutton"/>.
+* **Increase** <img src="../img/button/zoom-in.jpg" class="ms-docbutton"/> the map zoom
 
-* You can decrease the map zoom by using the zoom out icon <img src="../img/button/zoom-out.jpg" class="ms-docbutton"/>.
+* **Decrease** <img src="../img/button/zoom-out.jpg" class="ms-docbutton"/> the map zoom
 
-* You can switch to full screen view <img src="../img/button/full-screen.jpg" class="ms-docbutton"/>.
+* **Switch to full screen**  <img src="../img/button/full-screen.jpg" class="ms-docbutton"/> view
 
-* You can navigate through the extents used in a session by going back <img src="../img/button/back-extent.jpg" class="ms-docbutton"/> and forward <img src="../img/button/forward-extent.jpg" class="ms-docbutton"/> or to zoom to the maximum extent <img src="../img/button/max-extent.jpg" class="ms-docbutton"/>.
-
-3D Navigation
--------------
+* **Go back** <img src="../img/button/back-extent.jpg" class="ms-docbutton"/> to the last change 
 
-Based on CesiumJS, a 3D Navigator is implemented in [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) that allows you to display the map contents in 3D mode, particularly the layers will be overlaid on the globe.
+* **Go forward** <img src="../img/button/forward-extent.jpg" class="ms-docbutton"/> to the last change 
```

Comment:
**Go forward** <img src="../img/button/forward-extent.jpg" class="ms-docbutton"/> to the next map extent in the map navigation history